lib/translator/translation.cc

```diff
@@ -36,18 +37,26 @@ BeginResponse Translation::Begin(absl::Span<const uint8_t> scsi_cmd,
   pipeline_status_ = StatusCode::kSuccess;
   scsi_cmd_ = scsi_cmd;
 
+  uint32_t nsid = static_cast<uint32_t>(lun) + 1;
```

Comment:
why is this + 1

Reply:
Lun => nsid mapping has an offset of one because NSIDs start at 1 and LUNs are required to start at 0. I put this here so that going forward individual functions don't have to worry about LUN => nsid transformation and can just rely on this provided value.